core/src/main/scala/org/bitcoins/core/crypto/TransactionSignatureSerializer.scala

```diff
@@ -128,6 +136,9 @@ sealed abstract class TransactionSignatureSerializer {
         val sigHashBytes = Int32(hashType.num).bytes.reverse
 
         hashType match {
+          case SIGHASH_DEFAULT =>
+            sys.error(
+              s"SIGHASH_DEFAULT is only available in taproot signature serialization, got=${sigVersion}")
```

Comment:
Correct me if I'm wrong but I thought this mapped to `SIGHASH_ALL` in non-taproot signing

Reply:
Its unclear to me how the `0x00` sighash byte is treated in a pre-taproot world. I'm guessing the static test vectors i'm running will test this eventually.

Here is the pre-taproot sighash impl https://github.com/bitcoin/bitcoin/blob/master/src/script/interpreter.cpp#L1568